charts/images.yaml

```diff
@@ -30,15 +30,9 @@ images:
   repository: k8s.gcr.io/sig-storage/csi-node-driver-registrar
   tag: v1.3.0
 - name: csi-provisioner
-  sourceRepository: https://github.com/kubernetes-csi/external-provisioner
-  repository: k8s.gcr.io/sig-storage/csi-provisioner
-  tag: v1.6.0
-  targetVersion: "< 1.17"
-- name: csi-provisioner
-  sourceRepository: https://github.com/kubernetes-csi/external-provisioner
-  repository: k8s.gcr.io/sig-storage/csi-provisioner
-  tag: v2.1.2
-  targetVersion: ">= 1.17"
+  sourceRepository: https://github.com/kubernetes-sigs/alibaba-cloud-csi-driver
+  repository: registry.cn-hangzhou.aliyuncs.com/acs/csi-provisioner
+  tag: v3.0.0-3f86569-aliyun
```

Comment:
We cannot update to external-provisioner@v3.0.0 just like this currently because of the reasons listed in https://github.com/gardener/gardener-extension-provider-aws/pull/406#discussion_r717436048.
And what is the difference between this alicloud external-provisioner and the community maintained one?

Reply:
Yes, I agree. I was just adding it for testing purpose. On clusters with k8s version < 1.17, this csi-provisioner will fail, while with k8s version >= 1.17, cloud disks are successfully provisioned.